sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Config/SingleMessageOptions.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Messaging.ServiceBus;
+using Microsoft.Azure.WebJobs.ServiceBus;
+
+namespace Microsoft.Azure.WebJobs.ServiceBus
+{
+    /// <summary>
+    /// Configuration options for functions that bind to a single <see cref="ServiceBusReceivedMessage"/>.
+    /// </summary>
+    public class SingleMessageOptions
+    {
+        /// <summary>
+        /// Gets or sets the maximum duration within which the lock will be renewed automatically. This
+        /// value should be greater than the longest message lock duration; for example, the LockDuration Property.
+        /// The default value is 5 minutes.
+        /// </summary>
+        public TimeSpan MaxAutoLockRenewalDuration
+        {
+            get => _maxAutoRenewDuration;
+
+            set
+            {
+                Argument.AssertNotNegative(value, nameof(MaxAutoLockRenewalDuration));
+                _maxAutoRenewDuration = value;
+            }
+        }
+        private TimeSpan _maxAutoRenewDuration = TimeSpan.FromMinutes(5);
+
+        /// <summary>Gets or sets the maximum number of concurrent calls to a function. Note each call
+        /// would be passing a different message. The default is 16 times the return value of
+        /// <see cref="Utility.GetProcessorCount"/>.
```

Comment:
`Utility` is internal and not something that customers would have awareness of.   Maybe revise to something like "...  The default is 16 times the count of available processors" ?